geonode/upload/models.py

```diff
@@ -235,7 +237,7 @@ def get_import_url(self):
             return None
 
     def get_detail_url(self):
-        if self.resource and self.state == enumerations.STATE_PROCESSED:
+        if self.resource and self.resource.processed and self.state == enumerations.STATE_PROCESSED:
```

Comment:
in this case the `self.state == enumerations.STATE_PROCESSED` is redundant since is already inside the processed property

Reply:
Done